cpp/src/rolling/grouped_rolling.cu

```diff
@@ -115,6 +128,14 @@ std::unique_ptr<column> grouped_rolling_window(table_view const& group_keys,
   CUDF_EXPECTS((default_outputs.is_empty() || default_outputs.size() == input.size()),
                "Defaults column must be either empty or have as many rows as the input column.");
 
+  // Detect and bypass fully UNBOUNDED windows.
+  if (can_optimize_unbounded_window(preceding_window_bounds.is_unbounded,
```

Comment:
It seems a bit awkward that `window_bounds` has a member `is_unbounded`, while `range_window_bounds` has a method `is_unbounded()`. Should we file an issue to rectify that? The method seems better.

Reply:
This is mostly used internally. I'll fix it here.